utils/serde/src/main/java/io/apicurio/registry/utils/serde/AbstractKafkaSerDe.java

```diff
@@ -109,6 +107,9 @@ protected void configure(Map<String, ?> configs) {
         }
         if (idHandler == null) {
             Object idh = configs.get(REGISTRY_ID_HANDLER_CONFIG_PARAM);
+            if (idh == null) {
```

Comment:
Should not be needed ... getIdHandler() takes care of null ...

Reply:
Ah yes, forgot to revert that change.  On it.